choreographer/src/main/java/eu/arrowhead/core/choreographer/ChoreographerController.java

```diff
@@ -189,7 +233,7 @@ public void removeActionPlanById(@PathVariable(value = PATH_VARIABLE_ID) final l
             @ApiResponse(code = HttpStatus.SC_INTERNAL_SERVER_ERROR, message = CoreCommonConstants.SWAGGER_HTTP_500_MESSAGE)
     })
     @GetMapping(path = PLAN_MGMT_URI, produces = MediaType.APPLICATION_JSON_VALUE)
-    @ResponseBody public List<ChoreographerPlanResponseDTO> getChoreographerActionPlans(
+    @ResponseBody public List<ChoreographerPlanResponseDTO> getPlans(
```

Comment:
need a ChoreographerPlanListResponseDTO that contains this list and the total count of plans